src/main/java/com/saucelabs/bamboo/sod/action/PostBuildAction.java

```diff
@@ -82,6 +83,10 @@ public void println(String x) {
             try {
                 SauceTunnelManager sauceTunnelManager = SauceConnectFourManagerSingleton.getSauceConnectFourTunnelManager();
                 String options = customVariableContext.substituteString(config.getSauceConnectOptions(), buildContext, null);
+                if (config.useGeneratedTunnelIdentifier()) {
+                    String tunnelIdentifier = customVariableContext.getVariables(buildContext).get(TEMP_TUNNEL_ID);
```

Comment:
Can you change this to be SODKeys.TEMP_TUNNEL_ID to be explicit where its coming from?